CHANGELOG.md

```diff
@@ -96,7 +96,7 @@
 - [Added warning handling to `Table.aggregate`][3349]
 - [Improved performance of `Table.aggregate` and full warnings implementation]
   [3364]
-- [Implemented `Text.reverse`][3377]
+- [Implemented `Text.reverse`][3377, 3381]
```

Comment:
I don't think that Markdown supports that https://github.com/enso-org/enso/blob/bfb2af6b4d98b92499ba122bd5c368c9aedeb59c/CHANGELOG.md 

You can leave the link (change down below), and keep this line as is.

Reply:
My bad! Fixed.